src/Context/category-context.js

```diff
@@ -0,0 +1,23 @@
+import { createContext,useState,useContext,useEffect } from "react";
+import axios from "axios";
+
+
+const CategoryContext =createContext();
+const CategoryProvider =({children})=>{
+    const [categories, setCategories] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const { data } = await axios.get("/api/categories");
+        console.log(data);
```

Comment:
Can remove this console.log it is repeatedly showing up on the console and you can pass an empty dependency array so that the categories get loaded on the initial render.